### PR TITLE
New-CmAzService - initial implementation

### PR DESCRIPTION
**Pull request by tm-luke**

### CODE REVIEW

File: Cloudmarque.Azure/Public/Deploy/Core/New-CmAzService.ps1 [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+        Write-CommandStatus -CommandName $MyInvocation.MyCommand.Name
+
+        $SettingsObject = Get-Settings -SettingsFile $SettingsFile -SettingsObject $SettingsObject -CmdletName (Get-CurrentCmdletName -ScriptRoot $PSCommandPath)
+
+        if ($PSCmdlet.ShouldProcess((Get-CmAzSubscriptionName), "Deploy Service Locator Tag/s")) {
+
+            $SettingsObject.services | ForEach-Object -Parallel {
+                
+                $service = $_
+                $serviceTag = @{ "cm-service" = $_.value }
```
Comment (deepend-dev) [inline Cloudmarque.Azure/Public/Deploy/Core/New-CmAzService.ps1:48]: $service.value

File: Cloudmarque.Azure/Resources/Project/Integration/services.yml [hunk 2]
```diff
@@ -0,0 +1,31 @@
+services:
+- value: NC-Test-Value1
+  resourceGroupIds: [/subscriptions/e83613e4-e102-4d93-b874-8019611ab34d/resourceGroups/lukefapi/providers/Microsoft.Web/sites/lukefapi]
```
Comment (deepend-dev) [inline Cloudmarque.Azure/Resources/Project/Integration/services.yml:3]: can we also give simple rg names as well?

File: Cloudmarque.Azure/Resources/Schema/New-CmAzService.Schema.json [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+    "type": "object",
+    "title": "New-CmAzService",
+    "description": "Settings Root.",
+    "required": [
+        "services"
+    ],
+    "properties": {
+        "services": {
+            "$id": "#/services",
+            "type": ["array"],
```
Comment (deepend-dev) [inline Cloudmarque.Azure/Resources/Schema/New-CmAzService.Schema.json:13]: probably dont need brackets

File: Cloudmarque.Azure/Resources/Schema/New-CmAzService.Schema.json [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+                        "$id": "#/services/items/anyOf/rgs",
+						"type": "object",
+						"required": [
+							"value",
+                            "resourceGroupIds"
+						],
+						"properties": {
+                            "value": {
+                                "$id": "#/services/items/anyOf/rgs/value",
+                                "type": ["string"],
```
Comment (deepend-dev) [inline Cloudmarque.Azure/Resources/Schema/New-CmAzService.Schema.json:30]: brackets are not required

File: Cloudmarque.Azure/Public/Deploy/Core/New-CmAzService.ps1 [hunk 5]
```diff
@@ -0,0 +1,62 @@
+function New-CmAzService {
+
+    <#
+        .Synopsis
+         Helper function for tagging services using Set-CmAzTag.
+
+        .Description
+         Utilises Set-CmAzTag in order to tag specific resources or resource group with the "cm-service" tag and a specified value.
```
Comment (deepend-dev) [inline Cloudmarque.Azure/Public/Deploy/Core/New-CmAzService.ps1:8]: as we are not using set-cmaztag , this should change

**Review by deepend-dev — CHANGES_REQUESTED** [review]

File: Cloudmarque.Azure/Resources/Schema/New-CmAzService.Schema.json (hunk 4 above)
Comment (tm-luke) [reply Cloudmarque.Azure/Resources/Schema/New-CmAzService.Schema.json:30]: fixed

**Review by deepend-dev — DISMISSED** [review]

**Pull request merged**